src/auro-datepicker.js

```diff
@@ -345,6 +347,41 @@ export class AuroDatePicker extends LitElement {
     this.calendar.year = Number(this.centralDate.substring(6));
   }
 
+  /**
+   * Sends event notifying that the calendar popover has been opened.
+   * @private
+   * @returns {void}
+   */
+  notifyCalendarOpened() {
+    this.dispatchEvent(new CustomEvent('auroDatePicker-calendarOpened', {
+      bubbles: true,
+      composed: true,
+      detail: {
+        month: this.calendar.month,
+        year: this.calendar.year,
+        numCalendars: this.calendar.numCalendars,
+      },
```

Comment:
Let's follow the same naming pattern as dropdown.

```suggestion
    this.dispatchEvent(new CustomEvent('auroDatePicker-toggled', {
      bubbles: true,
      composed: true,
      detail: {
        exanded: this.dropdown.isPopoverVisible
      },
```

I don't think we should include the other details in this event.

For you to extract the month, year and, numCalendars you have to query `event.detail.month` etc. anyhow. It's just as easy to do `datepicker.calendar.month` etc.